### PR TITLE
Add widevine third party

### PR DESCRIPTION
**Pull request by clebeaupin**

### CODE REVIEW

File: script/create-dist [hunk 1]
```diff
@@ -110,9 +116,15 @@ INCLUDE_DIRS = [
...
   'cc',
   'chrome/browser/ui/libgtk2ui',
+  'components/cdm',
   'components/devtools_discovery',
   'components/devtools_http_handler',
   'components/os_crypt',
+  'components/component_updater',
+  'components/content_settings',
+  'components/nacl',
+  'components/omnibox',
```
Comment (deepak1556) [inline script/create-dist]: Any reason for including `content_settings`, `nacl` and `omnibox` components ?

Comment (clebeaupin) [inline script/create-dist]: You're right I don't know if they are required. I will try to compile electron without these dependencies and tell you if we need it or not

Comment (clebeaupin) [inline script/create-dist]: I have removed these components. I was able to compile electron without these components

**Comment by clebeaupin** [comment]
There is an issue during compilation on Windows.
There a lot warning about size_t conversion to uint32 in file vendor/media/cdm/stub/stub_cdm.cc.
This could be fixed by casting those types using base::checked_cast<uint32_t> function.
It has been fixed in recent versions of chromium

**Comment by zcbenz** [comment]
This looks good to me, but currently we are having issues on our OS X build machines, so I'm holding no to this for a few weeks.

> There is an issue during compilation on Windows.

You are probably using a newer VS, the supported compiler for Chrome 46 is VS 2013.

**Comment by clebeaupin** [comment]
I am using VS 2013 community on  Windows10 64 bits.
I will push a patch for stub_cdm.cc this afternoon

**Comment by clebeaupin** [comment]
Hi, 
I have resolved the conflicts. Do you think you can merge this request before releasing a new version of libchromiumcontent ?

Best regards

**Comment by zcbenz** [comment]
I'll merge this after I merged the desktopCapturer pull requests.

**Comment by zcbenz** [comment]
@clebeaupin Can you make a pull request that adds the widevine in Electron? Otherwise it is impossible for me to verify whether this actually works.

**Comment by clebeaupin** [comment]
ok

**Comment by clebeaupin** [comment]
Done: https://github.com/atom/libchromiumcontent/pull/165

**Comment by zcbenz** [comment]
Sorry if I didn't make it clear, you can't get EME support for free by adding those libraries into libchromiumcontent, we have to add corresponding code in Electron to make everything work. What I need is a pull request that adds the code in Electron.

**Comment by clebeaupin** [comment]
Yes I have the code for electron but I wanted to push it after you release the new version of libchromiumcontent to make sure it works

**Comment by zcbenz** [comment]
If you haven't tested the local build of libchromiumcontent with Electron, then I'm pretty sure it won't work.  I can not just merge the pull request and pretend everything will work.

**Comment by clebeaupin** [comment]
Ok I understand. I will create a pull request for electron including widevine

**Comment by clebeaupin** [comment]
As you requested, I have created  a pull request to integrate widevine third party into electron https://github.com/atom/electron/pull/3830

**Comment by zcbenz** [comment]
@clebeaupin Thanks! I'll start merging the pull requests soon after finishing some urgent tasks.

**Comment by zcbenz** [comment]
Merged with #168.

**Pull request closed without merging**